web/src/settings_panel_menu.js

```diff
@@ -18,7 +18,7 @@ export function mobile_deactivate_section() {
 }
 
 function two_column_mode() {
-    return $("#settings_overlay_container").css("--single-column") === undefined;
+    return $("html").css("--query-md-min-active") === undefined;
```

Comment:
`--query-md-min-active` is a confusing name, especially if we intend to extend this to some kind of convention: it’s not clear from the name whether “active” means `width < breakpoint` or `width >= breakpoint`.

Also, I think the point of setting a variable on `#settings_overlay_container` was to centralize the logic that governs the two-column settings panel in CSS. Now if someone were to change that logic (e.g. by changing the breakpoint, or migrating to a container query), they have to remember to change it in both CSS and JS.

Reply:
I agree it's a confusing name, for exactly the reasons you list. Let me think on this, and probably pull off this part of the PR so it doesn't hold up the rest.

The problem with `#settings_overlay_container` is that it's used with other parts of the modal that are still two columns, even with `< $md_min` in effect.